test/unit/ClauseResultsHelper.test.ts

```diff
@@ -162,4 +162,40 @@ describe('ClauseResultsHelpers', () => {
       expect(ret).toBeNull();
     });
   });
+
+  describe('isSupplementalDataElementStatement', () => {
+    let supplementalDataElements: any;
+    beforeEach(() => {
+      // Use the supplementalData from this measure for these tests.
+      const measure: fhir4.Measure = getJSONFixture('measure/simple-measure.json');
```

Comment:
IMO there's no need to use a fixture here since the input is so small. can just hardcode it like so:

```ts
const result = ClauseResultsHelpers.isSupplementalDataElementStatement(
  [
    {
      criteria: {
        language: 'text/cql.identifier'
      }
    }
  ],
  'SDE'
);
```

Reply:
The only thing I think I need to add is criteria.expression